tests/conftest.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from django.contrib.auth.models import AnonymousUser
+from django.contrib.sessions.middleware import SessionMiddleware
+from django.test import RequestFactory
+
+from factories import (UserFactory, InstitutionFactory)
+
+
+@pytest.fixture
+def user(uai_number):
+    user = UserFactory(
+        id=1,
```

Comment:
I don't think that it is a good practice to fix the id.

Reply:
Fixed by a37b1fd